dispatch/dispatch.go

```diff
@@ -362,6 +364,10 @@ func (ag *aggrGroup) fingerprint() model.Fingerprint {
 	return ag.labels.Fingerprint()
 }
 
+func (ag *aggrGroup) GroupKey() uint64 {
+	return uint64(ag.labels.Fingerprint() ^ ag.routeFP)
```

Comment:
I was thinking the same about the Fingerprint, but sticked with the `grouopKey` as it was already in the webhook. This is up to you guys :)